proto/api/v2/compliance_profile_service.proto

```diff
@@ -54,10 +78,17 @@ service ComplianceProfileService {
         };
     }
 
-    // ListComplianceProfiles returns profiles matching given query
-    rpc ListComplianceProfiles (RawQuery) returns (ListComplianceProfilesResponse) {
+    // ListComplianceProfiles returns profiles matching given request
+    rpc ListComplianceProfiles (ProfilesForClusterRequest) returns (ListComplianceProfilesResponse) {
+        option (google.api.http) = {
+            get: "/v2/compliance/cluster/profiles"
```

Comment:
I noticed the API docs don't flag the `clusterId` as required. Should it?

![image](https://github.com/stackrox/stackrox/assets/94937504/35ca6dd3-72f7-49dd-8e5f-49ed4969499a)

Other APIs flag required arguments in the path:

![image](https://github.com/stackrox/stackrox/assets/94937504/32c3fe17-62cf-4f0b-822a-7423d9899fb3)